tibanna/cw_utils.py

```diff
@@ -121,7 +122,7 @@ def plot_metrics(self, instance_type, directory='.', top_content=''):
             'max_disk_space_used_GB': (max_disk_space_used_GB_chunks_all_pts, 1),
             'max_mem_utilization_percent': (max_mem_utilization_percent_chunks_all_pts, 1),
             'max_disk_space_utilization_percent': (max_disk_space_utilization_percent_chunks_all_pts, 1),
-            'max_cpu_utilization_percent': (max_cpu_utilization_percent_chunks_all_pts, 5)
+            'max_cpu_utilization_percent': (max_cpu_utilization_percent_chunks_all_pts, 1)
```

Comment:
We are now using a 1 min resolution for CPU utilization. Not sure why we used a 5 min resolution before.